addons/crm/data/crm_lead_demo.xml

```diff
@@ -396,9 +396,6 @@ Andrew</p>]]></field>
             <field name="probability">30.0</field>
             <field name="partner_id" ref="base.res_partner_2"/>
             <field name="email_from">virginie@agrolait.com</field>
-            <field name="country_id" ref="base.be"/>
-            <field name="city">Wavre</field>
-            <field name="street">Rue de Namur 69</field>
```

Comment:
If we remove those values, can you confirm me that the system will use the country, city, street set on the associated client (i.e: `Deco Addict ‒ US12345673`)?

Reply:
Hello @julien-banken 
Yes I can confirm that 
Here is a screenshot from my runbot
https://i.imgur.com/gJtJzi1.png
Thanks :blush: